spec/controllers/games_controller_spec.rb

```diff
@@ -2,4 +2,29 @@
 
 RSpec.describe GamesController, type: :controller do
 
+  describe 'games#create action' do
+
+    it "should successfully create a game in the database" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+
+      post :create, game: {name: 'Testgame'}
+
+      game = Game.last
+      expect(game.name).to eq("Testgame")
+    end
+
+    it "should add the current user as white player when creating game" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+
+      post :create, game: {name: 'Testgame',
+                            player_white_id: user.id}
```

Comment:
@ilyakrasnov: I know this one is a problem. I'm having a hard time formulating controller tests with the information being passed via forms. Could use some guidance here, and it's also on the agenda to discuss more thoroughly with my mentor midweek.